src/Hintman/Comment.hs

```diff
@@ -0,0 +1,45 @@
+{-# LANGUAGE DataKinds #-}
+
+{- | This module contains functions to construct and submit comments.
+-}
+
+module Hintman.Comment
+       ( submitReview
+       ) where
+
+import Data.Aeson (encode)
+import GitHub.Auth (Auth (..))
+import GitHub.Data.Request (CommandMethod (Post), RW (RW), Request, command)
+import GitHub.Request (executeRequest)
+
+import Hintman.Core.PR (Owner (..), PrNumber (..), Repo (..))
+import Hintman.Core.Review (Comment (..), PullRequestReview (..), ReviewEvent (..), ReviewResponse)
+
+
+{- | This function submits PR review according to the following API endpoint:
+
+* https://developer.github.com/v3/pulls/reviews/#create-a-pull-request-review
+-}
+submitReview
+    :: Owner
+    -> Repo
+    -> PrNumber
```

Comment:
In the PR I'm working at I'm putting all this into one `PrInfo` data type, so no need to have all of these types separately.

Reply:
Nice! This function can be changed then to a simpler version :slightly_smiling_face: